clang/include/clang/CConv/ProgramVar.h

```diff
@@ -135,7 +137,8 @@ class FunctionParamScope : public ProgramVarScope {
   }
 
   bool operator==(const ProgramVarScope &O) const {
-    if (const FunctionParamScope *FPS = clang::dyn_cast<FunctionParamScope>(&O)) {
+    if (const FunctionParamScope *FPS =
```

Comment:
Instead of the line break you could make it shorter with `auto`. Type is redundant anyways since it's in the cast.

Reply:
Done.